src/web/blocks.rs

```diff
@@ -28,19 +29,20 @@ pub struct BlockShallowDTO {
     timestamp: Timestamp,
     block_hash: SerScaleHex<Hash>,
     transactions: u32,
-    rejected_transactions: u32,
+    // rejected_transactions: u32,
+    signature: SignaturesOf<CommittedBlock>,
 }
-
-impl TryFrom<BlockValue> for BlockShallowDTO {
+impl TryFrom<VersionedCommittedBlock> for BlockShallowDTO {
     type Error = color_eyre::Report;
-
-    fn try_from(block: BlockValue) -> Result<Self> {
+    fn try_from(block: VersionedCommittedBlock) -> Result<Self> {
+        let committed_block = block.into_v1();
```

Comment:
```suggestion
        let block = block.into_v1();
```

Renaming here is redundant and produces changes in below lines without reason.